core/src/banking_stage/transaction_scheduler/thread_aware_account_locks.rs

```diff
@@ -157,14 +168,20 @@ impl ThreadAwareAccountLocks {
                     ThreadSet::any(self.num_threads)
                 }
             }
-            (Some(write_locks), None) => ThreadSet::only(write_locks.thread_id),
```

Comment:
nit: i wonder why this arm is reordered?

Reply:
I probably just screwed it up somehow, re-ordered to original order now. But still left the unreachable arm at the bottom!

0f4d60d7e775c1472f7a540797875508b2aa2008